templates/kubernetes/base/deployment.yml

```diff
@@ -66,7 +66,7 @@ spec:
             valueFrom:
                 secretKeyRef:
                   name: cf-keypair
-                  key: secret_key
+                  key: private_key
```

Comment:
this should be matching the k8s secrets  https://github.com/commitdev/zero-aws-eks-stack/blob/f7aab988419adbed1e09b9d426eed96ec9ea59ea/templates/kubernetes/terraform/modules/kubernetes/backend_service.tf#L20

Reply:
He has a new PR which changes this name.